client/homepage/style.scss

```diff
@@ -13,18 +13,41 @@
 		width: calc(50% - 12px);
 		margin: 0;
 
-		@include breakpoint('<782px') {
+		@include breakpoint( '<782px' ) {
 			width: 100%;
 			position: inherit;
 			top: auto;
 		}
+
+		& > * {
+			margin-bottom: 24px;
+		}
 	}
 
-	@include breakpoint('<782px') {
+	@include breakpoint( '<782px' ) {
 		margin-left: -16px;
 		margin-right: -16px;
 		flex-direction: column-reverse;
 	}
+
+	.components-card {
+		.components-card__header.is-size-large {
+			display: grid;
+			//align-items: center;
+			min-height: 63px;
+			padding: 13px 16px;
```

Comment:
We should stick with `$gap` and `$gap-small` if possible.

Reply:
Is it possible to use the [`Card`](https://wordpress.github.io/gutenberg/?path=/story/components-card--default) component here? Then we don't have to worry about things like padding values.

Reply:
I'm already using the `@wordpress/components` `Card` component. But, the resulting alignment without tweaking things, differs from both the `@woocommerce/components` `Card` component and the design Figma.

I've updated the code to use the gap variables, as suggested.